keras_retinanet/preprocessing/generator.py

```diff
@@ -167,4 +167,34 @@ def next(self):
             labels_batch[index, ...]     = labels
             regression_batch[index, ...] = regression
 
-        return image_batch, [regression_batch, labels_batch]
+        return [regression_batch, labels_batch]
+
+    def compute_batches(self, group_index):
```

Comment:
The name is slightly misleading. As it is it sounds like it computes the division into batches, but it computes the input and output for a batch.

Some possible alternatives:
 * `compute_batch_input_output`
 * `compute_input_output_batch`
 * `compute_input_output`
 * `compute_batch_blobs`

Reply:
Renamed to `compute_input_output`.